src/main/scala/diplomaticobjectmodel/OMProcessor.scala

```diff
@@ -0,0 +1,16 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.tile
+
+import freechips.rocketchip.diplomacy.ResourceBindingsMap
+import freechips.rocketchip.diplomaticobjectmodel.DiplomaticObjectModelUtils
+import freechips.rocketchip.diplomaticobjectmodel.model.OMComponent
+import freechips.rocketchip.util.ElaborationArtefacts
+
+case object OMProcessor {
+  def process(resourceBindingsMap: ResourceBindingsMap): Unit = {
```

Comment:
Why `process`?

Reply:
I had pipeline() before and was asked by a reviewer in a meeting to change this to process.

Reply:
What is a "process" in this context? Not "processor"?

Reply:
Not a processor (cpu/core/micro engine). This is the function that runs the pipeline that creates the OM.